### PR TITLE
feat: add delay property to text-generate-effect component

### PR DESCRIPTION
**Pull request by nethriis**

### CODE REVIEW

**Comment by nethriis** [comment]
How can I add a demo ?

**Comment by nathan-de-pachtere** [comment]
> How do I add a demo?

- Look how it's done inside https://github.com/unovue/inspira-ui/blob/main/content/2.components/text-generate-effect.md
- Add a folder named text-generate-effect inside https://github.com/unovue/inspira-ui/tree/main/components/content/inspira/examples
- Move the https://github.com/unovue/inspira-ui/blob/main/components/content/inspira/examples/TextGenerateDemo.vue inside your new folder 
- Duplicate it, change it the way you want it
-  Update https://github.com/unovue/inspira-ui/blob/main/content/2.components/text-generate-effect.md

**Review by nathan-de-pachtere — APPROVED** [review]
Thanks for your effort! 💪

File: content/2.components/text-generate-effect.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ description: A cool text effect that fades in text on page load, one by one.
 | ---------- | --------- | -------- | --------------------------------------------------------- |
 | `words`    | `string`  | Required | The text to be displayed with the generating effect.      |
 | `duration` | `number`  | `0.7`    | The duration of the text generation animation in seconds. |
+| `delay`    | `number`  | `0.1`    | The delay before the text generation animation starts.    |
```
[the remark this answers is not in the document]
Comment (nethriis) [reply content/2.components/text-generate-effect.md]: Okay, I specified the time unit for the delay property.

**Review by nathan-de-pachtere — APPROVED** [review]

**Review by rahul-vashishtha — CHANGES_REQUESTED** [review]
Please format and lint the code using pnpm format & pnpm lint command.
Please fix if linting errors too.

**Comment by nethriis** [comment]
> Please format and lint the code using pnpm format & pnpm lint command. Please fix if linting errors too.

I think it must be good now.

**Review by rahul-vashishtha — APPROVED** [review]

**Pull request merged**